## Description: add some description
## Origin/Author: add some origin or author
## Bug: bug URL
Author: Michael Vogt <michael.vogt@ubuntu.com>
Description: Various default settings
 Changes in various defaults.  From what I can tell (these were originally
 from ubuntu, but I (the debian maintainer) will do my best to summarize
 them below).
 .
 * Desktop Horizontal virtual size of 2 instead of 4.
 * Sync to vblank: false by default.
 * Make an exception to focus prevention matching for PolicyKit windows.
 * Remove top-right hotcorner from scale plugin by default.
 * Increase drop-shadow radius from 8 to 9.
 * Set default decorator to /usr/bin/compiz-decorator.
 * Change default colors for faces in cube plugin.
 * Use "Smart" placement mode (2) by default for placement plugin.
 * Use "Rectangle" resize mode (2) by default for resize plugin.
 * Add exception to not fade notify-osd windows in fade plugin.
 .
 The following changes from the original Ubuntu patch were modified:
 * Use slightly more Debianish/Gnomeish colors for the cube face top.
 The following changes from the original Ubuntu patch were not included:
 * Add Ubuntu logo for top face of cube.
 The following changes were introduced to the Debian version of this patch:
 * Add Debian logo for top face of cube.
Origin: vendor, ubuntu (1:0.6.0+git20071006-0ubuntu1) or perhaps earlier.
Index: compiz-0.9.4/metadata/core.xml.in
===================================================================
--- compiz-0.9.4.orig/metadata/core.xml.in	2011-02-24 10:27:13.000000000 +0100
+++ compiz-0.9.4/metadata/core.xml.in	2011-03-17 17:12:02.028886771 +0100
@@ -138,7 +138,7 @@
 			<option name="focus_prevention_match" type="match">
 			<_short>Focus Prevention Windows</_short>
 			<_long>Focus prevention windows</_long>
-			<default>any</default>
+			<default>!(class=Polkit-gnome-authentication-agent-1)</default>
 			</option>
 		</group>
 		<group>
@@ -244,14 +244,14 @@
 			<option name="hsize" type="int">
 			<_short>Horizontal Virtual Size</_short>
 			<_long>Screen size multiplier for horizontal virtual size</_long>
-			<default>4</default>
+			<default>2</default>
 			<min>1</min>
 			<max>32</max>
 			</option>
 			<option name="vsize" type="int">
 			<_short>Vertical Virtual Size</_short>
 			<_long>Screen size multiplier for vertical virtual size</_long>
-			<default>1</default>
+			<default>2</default>
 			<min>1</min>
 			<max>32</max>
 			</option>
Index: compiz-0.9.4/plugins/scale/scale.xml.in
===================================================================
--- compiz-0.9.4.orig/plugins/scale/scale.xml.in	2011-02-24 10:27:13.000000000 +0100
+++ compiz-0.9.4/plugins/scale/scale.xml.in	2011-03-17 17:12:36.117055808 +0100
@@ -18,14 +18,14 @@
 			<option name="spacing" type="int">
 				<_short>Spacing</_short>
 				<_long>Space between windows</_long>
-				<default>10</default>
+				<default>68</default>
 				<min>0</min>
 				<max>250</max>
 			</option>
 			<option name="speed" type="float">
 				<_short>Speed</_short>
 				<_long>Scale speed</_long>
-				<default>1.5</default>
+				<default>2.4</default>
 				<min>0.1</min>
 				<max>50</max>
 				<precision>0.1</precision>
@@ -33,7 +33,7 @@
 			<option name="timestep" type="float">
 				<_short>Timestep</_short>
 				<_long>Scale timestep</_long>
-				<default>1.2</default>
+				<default>0.1</default>
 				<min>0.1</min>
 				<max>50</max>
 				<precision>0.1</precision>
@@ -46,7 +46,7 @@
 			<option name="opacity" type="int">
 				<_short>Opacity</_short>
 				<_long>Amount of opacity in percent</_long>
-				<default>75</default>
+				<default>100</default>
 				<min>0</min>
 				<max>100</max>
 			</option>
@@ -55,7 +55,7 @@
 				<_long>Overlay an icon on windows once they are scaled</_long>
 				<min>0</min>
 				<max>2</max>
-				<default>1</default>
+				<default>0</default>
 				<desc>
 					<value>0</value>
 					<_name>None</_name>
@@ -105,7 +105,7 @@
 			<option name="key_bindings_toggle" type="bool">
 			    <_short>Key Bindings Toggle Scale Mode</_short>
 			    <_long>Key bindings toggle scale mode instead of enabling it when pressed and disabling it when released.</_long>
-			    <default>false</default>
+			    <default>true</default>
 			</option>
 			<option name="button_bindings_toggle" type="bool">
 			    <_short>Button Bindings Toggle Scale Mode</_short>
@@ -115,9 +115,6 @@
 			<option name="initiate_edge" type="edge">
 				<_short>Initiate Window Picker</_short>
 				<_long>Layout and start transforming windows</_long>
-				<default>
-				<edge name="TopRight"/>
-				</default>
 				<allowed edgednd="true"/>
 			</option>
 			<option name="initiate_key" type="key">
@@ -141,6 +138,7 @@
 			<option name="initiate_all_key" type="key">
 				<_short>Initiate Window Picker For All Windows</_short>
 				<_long>Layout and start transforming all windows</_long>
+				<default>&lt;Super&gt;w</default>
 			</option>
 			<option name="initiate_group_edge" type="edge">
 				<_short>Initiate Window Picker For Window Group</_short>
@@ -171,7 +169,7 @@
 			<option name="show_desktop" type="bool">
 				<_short>Click Desktop to Show Desktop</_short>
 				<_long>Enter Show Desktop mode when Desktop is clicked during Scale</_long>
-				<default>true</default>
+				<default>false</default>
 			</option>
 	    </group>
 	</options>
Index: compiz-0.9.4/plugins/decor/decor.xml.in
===================================================================
--- compiz-0.9.4.orig/plugins/decor/decor.xml.in	2011-02-24 10:27:13.000000000 +0100
+++ compiz-0.9.4/plugins/decor/decor.xml.in	2011-03-17 17:12:02.028886771 +0100
@@ -20,7 +20,7 @@
 	    <option name="shadow_radius" type="float">
 		<_short>Shadow Radius</_short>
 		<_long>Drop shadow radius</_long>
-		<default>8.0</default>
+		<default>15.0</default>
 		<min>0.1</min>
 		<max>18.0</max>
 		<precision>0.1</precision>
@@ -47,13 +47,14 @@
 	    <option name="shadow_y_offset" type="int">
 		<_short>Shadow Offset Y</_short>
 		<_long>Drop shadow Y offset</_long>
-		<default>1</default>
+		<default>5</default>
 		<min>-16</min>
 		<max>16</max>
 	    </option>
 	    <option name="command" type="string">
 		<_short>Command</_short>
 		<_long>Decorator command line that is executed if no decorator is already running</_long>
+		<default>/usr/bin/compiz-decorator</default>
 	    </option>
 	    <option name="mipmap" type="bool">
 		<_short>Mipmap</_short>
Index: compiz-0.9.4/plugins/cube/cube.xml.in
===================================================================
--- compiz-0.9.4.orig/plugins/cube/cube.xml.in	2011-02-24 10:27:13.000000000 +0100
+++ compiz-0.9.4/plugins/cube/cube.xml.in	2011-03-17 17:12:02.028886771 +0100
@@ -86,18 +86,18 @@
 				<_short>Top</_short>
 				<_long>Color of top face of the cube</_long>
 				<default>
-					<red>0xffff</red>
-					<green>0xffff</green>
-					<blue>0xffff</blue>
+					<red>0xcdcd</red>
+					<green>0xbebe</green>
+					<blue>0x7070</blue>
 				</default>
 			</option>
 			<option name="bottom_color" type="color">
 				<_short>Bottom</_short>
 				<_long>Color of bottom face of the cube</_long>
 				<default>
-					<red>0xffff</red>
-					<green>0xffff</green>
-					<blue>0xffff</blue>
+					<red>0xcdcd</red>
+					<green>0xbebe</green>
+					<blue>0x7070</blue>
 				</default>
 			</option>
 			</subgroup>
@@ -112,6 +112,7 @@
 			<_short>Skydome Image</_short>
 			<_long>Image to use as texture for the skydome</_long>
 			<hints>file;image;</hints>
+			<default><value>/usr/share/gdm/themes/Human/ubuntu.png</value></default>
 			</option>
 			<option name="skydome_animated" type="bool">
 			<_short>Animate Skydome</_short>
Index: compiz-0.9.4/plugins/place/place.xml.in
===================================================================
--- compiz-0.9.4.orig/plugins/place/place.xml.in	2011-02-24 10:27:13.000000000 +0100
+++ compiz-0.9.4/plugins/place/place.xml.in	2011-03-17 17:12:02.032886795 +0100
@@ -19,7 +19,7 @@
 	    <option name="mode" type="int">
 		<_short>Placement Mode</_short>
 		<_long>Algorithm to use for window placement</_long>
-		<default>0</default>
+		<default>2</default>
 		<min>0</min>
 		<max>5</max>
 		<desc>
Index: compiz-0.9.4/plugins/resize/resize.xml.in
===================================================================
--- compiz-0.9.4.orig/plugins/resize/resize.xml.in	2011-02-24 10:27:13.000000000 +0100
+++ compiz-0.9.4/plugins/resize/resize.xml.in	2011-03-17 17:12:02.032886795 +0100
@@ -123,7 +123,7 @@
 	    <option name="mode" type="int">
 		<_short>Default Resize Mode</_short>
 		<_long>Default mode used for window resizing</_long>
-		<default>0</default>
+		<default>2</default>
 		<min>0</min>
 		<max>3</max>
 		<desc>
@@ -147,20 +147,20 @@
 		<_short>Border Color</_short>
 		<_long>Border color used for outline and rectangle resize modes</_long>
 		<default>
-		    <red>0x2f2f</red>
-		    <green>0x2f2f</green>
-		    <blue>0x4f4f</blue>
-		    <alpha>0x9f9f</alpha>
+		    <red>0xffff</red>
+		    <green>0xffff</green>
+		    <blue>0xffff</blue>
+		    <alpha>0x6464</alpha>
 		</default>
 	    </option>
 	    <option name="fill_color" type="color">
 		<_short>Fill Color</_short>
 		<_long>Fill color used for rectangle resize mode</_long>
 		<default>
-		    <red>0x2f2f</red>
-		    <green>0x2f2f</green>
-		    <blue>0x4f4f</blue>
-		    <alpha>0x4f4f</alpha>
+		    <red>0x0</red>
+		    <green>0x0</green>
+		    <blue>0x0</blue>
+		    <alpha>0x1e1e</alpha>
 		</default>
 	    </option>
 	    <option name="normal_match" type="match">
Index: compiz-0.9.4/plugins/fade/fade.xml.in
===================================================================
--- compiz-0.9.4.orig/plugins/fade/fade.xml.in	2011-02-24 10:27:13.000000000 +0100
+++ compiz-0.9.4/plugins/fade/fade.xml.in	2011-03-17 17:12:02.032886795 +0100
@@ -49,7 +49,7 @@
 	    <option name="window_match" type="match">
 		<_short>Fade windows</_short>
 		<_long>Windows that should be fading</_long>
-		<default>any</default>
+                <default>any &amp; !(title=notify-osd)</default>
 	    </option>
 	    <option name="visual_bell" type="bool">
 		<_short>Visual Bell</_short>
Index: compiz-0.9.4/plugins/opengl/opengl.xml.in
===================================================================
--- compiz-0.9.4.orig/plugins/opengl/opengl.xml.in	2011-02-24 10:27:13.000000000 +0100
+++ compiz-0.9.4/plugins/opengl/opengl.xml.in	2011-03-17 17:12:02.032886795 +0100
@@ -31,7 +31,7 @@
 	    <option name="lighting" type="bool">
 		<_short>Lighting</_short>
 		<_long>Use diffuse light when screen is transformed</_long>
-		<default>true</default>
+		<default>false</default>
 	    </option>
 	    <option name="sync_to_vblank" type="bool">
 		<_short>Sync To VBlank</_short>
